recipes/tqdm/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "4.7.2" %}
+
+package:
+  name: tqdm
+  version: {{ version }}
+
+source:
+  fn: tqdm-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/t/tqdm/tqdm-{{ version }}.tar.gz
+  md5: 63dcfa4910aa10ed7bf5c6e4a961ad7f
+
+build:
+  entry_points:
+    - tqdm=tqdm._main:main
+
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    # - setuptools
```

Comment:
Can you elaborate on the point? I was surprised that I got a build without problems after commenting out setuptools, but it seems to be unnecessary.